bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/TableUtils.java

```diff
@@ -125,6 +118,70 @@ public static void prepareAndExecuteStatement(PreparedStatement stmt, MapValueIm
         }
     }
 
+    private static void prepareAndExecuteStatement(PreparedStatement stmt, MapValueImpl<?, ?> data, int index,
```

Comment:
Same as above? Can we merge this with the caller method? Execution has to go through two switch cases it seems